RELEASE_NOTES.md

```diff
@@ -1,3 +1,7 @@
+# Version 0.5.9
```

Comment:
If we haven't had a release of 0.5.8, add this to that release, otherwise it's good here.

Reply:
Done.